docs/core/deploying/trimming/prepare-libraries-for-trimming.md

```diff
@@ -27,28 +27,37 @@ These instructions show how to enable and resolve static analysis warnings to pr
 > [!TIP]
 > Ensure you're using the .NET 6 SDK or later for these steps. They will not work correctly in previous versions.
```

Comment:
We may also want to note that the best results here will come from installing the .NET 7 preview SDK.